docs/installation.rst

```diff
@@ -59,3 +65,33 @@ additional functionality:
   .. code-block:: bash
 
       pip install pystac[orjson]
+
+Versions
+========
+
+To install a version of PySTAC that works with a specific versions of the STAC
+specification, install the matching version of PySTAC from the following table.
+
+.. list-table:: Versions
+   :widths: 50 50
+   :header-rows: 1
+
+   * - PySTAC
+     - STAC
+   * - 1.x
+     - 1.0.x
+   * - 0.5.x
+     - 1.0.0-beta.*
+   * - 0.4.x
+     - 0.9.x
+   * - 0.3.x
+     - 0.8.x
```

Comment:
I rendered the docs to make sure this came through propery ![image](https://user-images.githubusercontent.com/4806877/212161416-e374f4c7-5f81-433f-b9ee-32fea9775d1f.png)